docs/source/format/driver_manifests.rst

```diff
@@ -313,35 +316,39 @@ to control which directories will be searched for manifests, with the behavior b
     .. tab-item:: R
        :sync: r
 
-       Use ``adbc_driver(... , load_flags = adbc_load_flags())`` to pass options to the driver manager
+       Use ``adbc_driver(..., load_flags = adbc_load_flags())`` to pass options to the driver manager
        regarding how to locate drivers specified by manifest.
 
     .. tab-item:: Ruby
        :sync: ruby
 
-        The class ``ADBC::LoadFlags`` is a set of bitflags to control the directories to be searched. The flags are
-        * ``ADBC::LoadFlags::SEARCH_ENV` - search the environment variable ``ADBC_CONFIG_PATH``
-        * ``ADBC::LoadFlags::SEARCH_USER`` - search the user configuration directory
-        * ``ADBC::LoadFlags::SEARCH_SYSTEM`` - search the system configuration directory
-        * ``ADBC::LoadFlags::ALLOW_RELATIVE_PATHS`` - allow a relative path to be provided
-        * ``ADBC::LoadFlags::DEFAULT`` - default value with all flags set
+       The class ``ADBC::LoadFlags`` is a set of bitflags to control the directories to be searched. The flags are
 
-        These can be provided by using ``ADBC::Database#load_flags=``.
-        Passing the option ``load_flags`` as an option to ``AdbcDatabase`` (or via ``db_kwargs`` in ``adbc_driver_qmanager.dbapi.connect``) will
-        allow you to control the directories to be searched by using the value of the option as the bitmask for the load flag desired.
+       * ``ADBC::LoadFlags::SEARCH_ENV`` - search the environment variable ``ADBC_CONFIG_PATH``
+       * ``ADBC::LoadFlags::SEARCH_USER`` - search the user configuration directory
+       * ``ADBC::LoadFlags::SEARCH_SYSTEM`` - search the system configuration directory
+       * ``ADBC::LoadFlags::ALLOW_RELATIVE_PATHS`` - allow a relative path to be provided
+       * ``ADBC::LoadFlags::DEFAULT`` - default value with all flags set
+       These can be provided by using ``ADBC::Database#load_flags=``.
```

Comment:
you need a blank line before and after the bullet list

Reply:
Ugh, thanks. fb62f374402824bd7221df2668e15f0a659a5189

Reply:
and [c4e8e47](https://github.com/apache/arrow-adbc/pull/3180/commits/c4e8e47defaca737b5a5f525ecf5242761c620d9).